app/src/play/java/chat/rocket/android/analytics/GoogleAnalyticsForFirebase.kt

```diff
@@ -32,10 +34,15 @@ class GoogleAnalyticsForFirebase @Inject constructor(val context: Context) :
         })
     }
 
+    // WIDECHAT tracking BSSID
     override fun logMessageSent(event: SubscriptionTypeEvent, serverUrl: String) {
-        firebaseAnalytics.logEvent("message_sent", Bundle(2).apply {
+        val bssid = SharedPreferenceHelper.getString(Constants.CURRENT_BSSID, "none")
+        val loc_permission = SharedPreferenceHelper.getString(Constants.LOCATION_PERMISSION, "none")
+        firebaseAnalytics.logEvent("message_sent", Bundle(3).apply {
             putString("subscription_type", event.subscriptionTypeName)
-            putString("server", serverUrl)
+//            putString("server", serverUrl)
+            putString("wifi_bssid", bssid)
+            putString("location_permission", loc_permission)
```

Comment:
Why are we sending this in each message log?
Can we determine the granted/not-granted by setting wifi_bssid to some special value?

Reply:
We are setting it in the message event because we would like to track how our users are using our community wifi.  Do you have another suggestion for where to attach this parameter?  

It's probably a good suggestion to just populate the "wifi_bssid" field with the not-granted/do-not-show-again message, instead of creating a totally new field, "location_permission".  Is that what you mean?

Reply:
Yes, that's what I mean.